src/crypto/sign.js

```diff
@@ -301,3 +302,66 @@ export function verifyTransaction(transaction, secondPublicKey) {
 
 	return secondSignaturePresent ? verifyTransaction(transactionWithoutSignature) : verified;
 }
+
+/**
+ * @method encryptAES256CBCWithPassword
+ * @param plaintext utf8
+ * @param password
+ *
+ * @return {string}
+ */
+
+function encryptAES256CBCWithPassword(plaintext, password) {
+	const iv = crypto.randomBytes(16);
+	const passHash = getSha256Hash(password, 'utf8');
+	const cipherInit = crypto.createCipheriv('aes-256-cbc', passHash, iv);
+	const textCipher = cipherInit.update(plaintext, 'utf8');
+	const finishedCipher = Buffer.concat([textCipher, cipherInit.final()]);
+	const cipherText = ''.concat(iv.toString('hex'), '$', finishedCipher.toString('hex'));
+
+	return cipherText;
+}
+
+/**
+ * @method decryptAES256CBCWithPassword
+ * @param cipherText
+ * @param password
+ *
+ * @return {string}
+ */
+
+function decryptAES256CBCWithPassword(cipherText, password) {
+	const passHash = getSha256Hash(password, 'utf8');
+	const encryptedParts = cipherText.split('$');
+	const iv = Buffer.from(encryptedParts.shift(), 'hex');
+	const encryptedPass = Buffer.from(encryptedParts.join('$'), 'hex');
```

Comment:
If there's more than 1 member of `encryptedParts` left then something has gone wrong. Why not throw an error, or just use the first member? Otherwise, `Buffer.from` will effectively do one of these anyway but it will depend on where the `$` appears in the string.